<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Welcome Screen</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_User">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pixMap/userLogin1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pixMap/passwordLogin.png</pixmap>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Login">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pixMap/GradientLogin2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_AvisionR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>AvisionR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>191</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pixMap/eye.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Status...</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>lineEdit_User</zorder>
  <zorder>lineEdit_Password</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>commandLinkButton_Login</zorder>
  <zorder>label_AvisionR</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_Status</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Password</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Login</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_User</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Login</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
